python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt/dlt/resource.py

```diff
@@ -100,6 +109,30 @@ def extract_resource_metadata(
             for job in load_package.get("jobs", [])
             if job.get("table_name") == resource.table_name
         ]
+        rows_loaded = dlt_pipeline.last_trace.last_normalize_info.row_counts.get(
+            str(resource.table_name), None
+        )
+        if rows_loaded:
+            base_metadata["rows_loaded"] = MetadataValue.int(rows_loaded)
+        try:
+            with dlt_pipeline.sql_client() as client:
+                with client.execute_query(
+                    f"""
+                        SELECT
+                        count(*) as row_count
+                        FROM
+                        {resource.table_name}
+                    """,
+                ) as cursor:
+                    row_count_result = cursor.fetchone()
+        # Filesystem does not have a SQL client and table might not be found
+        except (SqlClientNotAvailable, DatabaseUndefinedRelation):
+            # TODO: Add logging
```

Comment:
```suggestion
            context.log.info("Unable to determine row_count metadata as SQL client unavailable")
```

Reply:
Don't know how I overlooked the fact that `context` has a logger. Will add this, thanks!